test/Microsoft.Azure.Devices.Edge.Test/EndToEnd.cs

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using NUnit.Framework;
+    using Serilog;
+
+    public class EndToEnd
```

Comment:
Do you expect all e2e tests to go in this file?

Reply:
That's my hope. Tests will need to become a lot shorter and easier to read than they are now, otherwise I'll probably need to split them up. But I hope they will be easy to read in the end, all in one file.